detection-rules/attachment_malicious_onenote_commands.yml

```diff
@@ -12,30 +12,25 @@ severity: "high"
 source: |
   type.inbound
   and any(attachments,
-      (
-          .file_extension in~ ("one") or
-          .file_extension in~ $file_extensions_common_archives
-      )
-      and any(file.explode(.),
-      any(
-          .flavors.yara, . == "onenote_file"
-           and
-            any(..scan.strings.strings,
```

Comment:
This was `..scan.strings.strings` because it was actually nested _inside_ the `any(.flavors.yara, ...)` loop!